lib/client/index.js

```diff
@@ -509,7 +509,7 @@ client.init = function init(serverSettings, plugins) {
     }
   }
 
-  function stopAlarm(isClient, silenceTime) {
+  function stopAlarm(isClient, silenceTime, notify) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2728688176/issues/source?bid=2452379&fileBranchId=3348623#l512)